### DISCUSSION TITLE
Required survey to change the state with permissions

### DISCUSSION THREAD

**Discussion by ValeMP** [Cotalker/documentation#349] [2. Questions and Answers] [answered]
Hola!

Estoy editando una rutina para que haga uso de un required survey para el cambio de estado. Observé que se puede agregar un permiso para que solo las personas que tengan el permiso puedan responder. Entonces, intente hacer el cambio de estado con el usuario que tenía asignado el permiso y no pude hacer el cambio.

Entonces, quería saber si me falto algún otro paso que añadir el permiso la seccion del required suvrey del workflow y al usuario.

Gracias!

**Comment by guillevalin** [comment, marked as answer]
@ValeMP Se descubrió un bug en esta funcionalidad, el cual se subió un hotfix en la tarde de hoy.

Cabe recordar que tanto el bot de la State Machine como el Usuario, deben tener los permissions para que se pueda realizar el cambio de estado.

Saludos!
